src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/String/Hashing.cs

```diff
@@ -138,6 +141,9 @@ public static unsafe int GetHashCodeOrdinalIgnoreCaseAscii(ReadOnlySpan<char> s)
 
         public static unsafe int GetHashCodeOrdinalIgnoreCase(ReadOnlySpan<char> s)
         {
+#if NET
```

Comment:
Worth a comment about NETFX perf here?

Reply:
Saying...?

Reply:
Never seen `NET` before. Is it a synonym for `NETCOREAPP` or something different?